uPortal-webapp/src/main/webapp/media/skins/respondr/common/images/layout_100.svg

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
```

Comment:
:thinking: it looks like some inkscape graphics metadata leaked into the SVG.
This data is useful if the image needs to be edited again, but adds attributes that browsers ignore and there add a little unneeded weight when serving.
Should these be scrubbed?

Reply:
removed with [svgo](https://github.com/svg/svgo)